plugins/queue/providers/rabbit_mq/lib/RabbitMQProvider.php

```diff
@@ -202,9 +204,25 @@ protected function closeChannel($channel)
 		{
 			$connTook = microtime(true) - $connStart;
 			$logStr = "Failed to close channel";
-			$this->writeToMonitor($logStr, $this->dataSourceUrl, self::RABBIT_ACTION_CHANNEL_OPERATIONS, $connTook, null, null, $e->getCode());
+			$this->writeToMonitor($logStr, $this->dataSourceUrl, self::RABBIT_ACTION_CLOSE_CHANNEL, $connTook, null, null, $e->getCode());
 			throw $e;
 		}
 
 	}
+
+	protected function closeConnection($connection)
+	{
+		$connStart = microtime(true);
```

Comment:
Misleading name - better use before, since we don't know when the connection really started.